DirectProgramming/DPC++FPGA/Tutorials/GettingStarted/fpga_template/fpga_compile.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.705
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{3F73FABA-60A1-44FE-92B0-280B68E688C5}") = "fpga_compile", "fpga_compile.vcxproj", "{525AC553-2703-448F-B5CA-3D2AAF275905}"
```

Comment:
```suggestion
Project("{3F73FABA-60A1-44FE-92B0-280B68E688C5}") = "fpga_template", "fpga_template.vcxproj", "{525AC553-2703-448F-B5CA-3D2AAF275905}"
```

Reply:
won't apply this change, as the .sln and .vcxproj files are being removed

Reply:
Yep, these have been removed today - so you can remove all .sln, .vcxproj files, as well as license.txt and third-party-programs.txt